.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,18 +11,21 @@ body:
 
         Do you need some help?
         ======================
-        The issue tracker is meant for feature requests and bug reports only. This isn't the best place for
-        support or usage questions. Questions here don't have as much visibility as they do elsewhere. Before
-        you ask a question, here are some resources to get help first:
+        The issue tracker is meant for bug reports only. This isn't the best
+        place for support or usage questions. Questions here don't have as much
+        visibility as they do elsewhere. Before you ask a question, here are
+        some resources to get help first:
 
         - Read the docs: https://reactrouter.com
         - Check out the list of frequently asked questions: https://reactrouter.com/start/faq
         - Explore examples: https://reactrouter.com/start/examples
+        - Ask in chat: https://rmx.as/discord
```

Comment:
Removed the old Reactiflux discord link in favor of the Remix discord, and moved it above stack overflow